packages/experimental/NativeButton/src/NativeButton.types.ts

```diff
@@ -0,0 +1,48 @@
+import { ImageURISource } from 'react-native';
+
+export const nativeButtonName = 'NativeButton';
+
+export interface NativeButtonProps {
+  /*
+   * Text to show on the Button.
+   */
+  title?: string;
+  /*
+   * Name of the icon to show on the Button.
+   */
+  image?: ImageURISource;
+  /*
+   * Should the button image be tinted
+   */
+  isImageTinted?: boolean;
+  /*
+   * ButtonStyle wrapped enum
+   */
+  buttonStyle?: NativeButtonStyle;
+  /*
+   * A button can be enabled or disabled.
+   */
+  isEnabled?: boolean;
+  /*
+   * A callback to call on button click event
+   */
+  onPress?: () => void;
+}
+
+export interface NativeButtonTokens {
+  /*
+   * Button background color
+   */
+  accentColor?: string;
```

Comment:
Can we add a comment saying this token is a no-op on iOS? Or split this to separate iOS and macOS specific token files?

Reply:
Given there's only one token defined, I'll add a comment to it for now.